letsencrypt-apache/letsencrypt_apache/tls_sni_01.py

```diff
@@ -1,4 +1,5 @@
-"""ApacheDVSNI"""
+"""A TLS-SNI-01 authenticator for Apache"""
```

Comment:
So this technically isn't an authenticator since it doesn't implement the `IAuthenticator` interface. Perhaps we should avoid using the term and just call it "A class that performs TLS-SNI-01 challenges for Apache"?

There are a few other places, with varying importance, where `authenticator` was used to refer to this class as well.